r/R/dotplot.R

```diff
@@ -1,35 +1,84 @@
+#' Title
+#'
+#' @param req {body: {
+#'               useMarkerGenes: True/False determines whether to use marker genes or predefined genes
+#'               numberOfMarkers: Int. Number of marker genes to use
+#'               customGenesList: List of Strings. List of marker genes to use
+#'               groupBy: Cellsets to show in the plot. Determines whether to show Louvain/Samples/Custom
+#'               filterBy: Cellsets to subset the experiment with.
+#'               isFilterByAll: Bool value. If true, the experiment will be subsetted to all the cellSets ids.
+#'              }
+#'            }
+#' @param data
+#'
+#' @return
+#' @export
+#'
+#' @examples
 runDotPlot <- function(req, data) {
-  type <- req$body$type
-  req_genes <- req$body$genes
+  useMarkerGenes <- req$body$useMarkerGenes
```

Comment:
```suggestion
  use_marker_genes <- req$body$useMarkerGenes
```

check elsewhere that not using camelCase except for function names as previously discussed